stdlib/builtins.pyi

```diff
@@ -81,6 +79,14 @@ _T4 = TypeVar("_T4")
 _T5 = TypeVar("_T5")
 _TT = TypeVar("_TT", bound="type")
 _TBE = TypeVar("_TBE", bound="BaseException")
+_SupportsNextT = TypeVar("_SupportsNextT", bound=SupportsNext[Any], covariant=True)
+_SupportsAnextT = TypeVar("_SupportsAnextT", bound=SupportsAnext[Any], covariant=True)
+
+class _Iterable2(Protocol[_T_co]):
```

Comment:
Why not `HasIter`?

Reply:
I've renamed to protocols (as `Supports...`). Do you think it would make sense to add them to `_typeshed`?